languages/typescript/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM node:alpine
 LABEL author="1Computer1"
 
-RUN yarn global add typescript @types/node
+RUN npm install --global --save-dev @swc/core @swc/register
```

Comment:
Does `--no-save` save any space here?

Reply:
I've honestly taken that command straight from their documentation. The `--save-dev` parameter is probably not even necessary here as it doesn't make a difference in this context. `--no-save` didn't seem to make a difference in size as it presumably just adds them to a global package-lock.
I also deliberately chose npm here instead of yarn as the swc-node binary for some reason wasn't available after a global install with yarn but I also didn't debug now where exactly yarn puts these packages.